data-platform/core-converged-db/oracle-database-23c-new-features/OracleJDBC23cNewFeatures/src/main/java/swierzy/oraclejdbc23newfeatures/QuerySubscriber.java

```diff
@@ -0,0 +1,46 @@
+package swierzy.oraclejdbc23newfeatures;

```

Comment:
@oheimburger

Reply:
Change the package name to `demo.jdbc23newfeatures`.